sechub-commons-model/src/test/java/com/daimler/sechub/commons/model/SecHubFindingTest.java

```diff
@@ -0,0 +1,77 @@
+package com.daimler.sechub.commons.model;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class SecHubFindingTest {
+
+    @Test
```

Comment:
I also thought about doing this as a parameterized test - but the class was not available. I had not the time to investigate. So I dropped the idea and just implemented normal tests...
After reading your comment I investigated again - and found the reason... 

Spring boot test starter seems to automatically add the jupiter dependencies for parametrized tests - in our commons model we have not this and must define the version by our own.

I will add the gradle dependency and change tests because more readable and easier to maintian.